web/src/pages/shoppingBasket.js

```diff
@@ -11,6 +11,7 @@ class ShoppingBasket extends Component {
         this.state = {
             numberOfSelectedItems: 0,
             totalPrice: 0,
+            ItemInfo: [],
```

Comment:
Since `ShoppingBasket` also managing data that are related to item info, why don't you manage these kinds of data in one place? Not distribute to `ShoppingBasket` and `ItemList`